src/main/java/edu/cornell/kfs/module/purap/jaggaer/supplier/xml/JaggaerBuilder.java

```diff
@@ -50,13 +53,24 @@ public static Active buildActive(String activeString) {
         return buildActive(activeString, null);
     }
     
+    public static Active buildActive(boolean isActive, JaggaerBooleanToStringTyoe jaggaerBooleanToStringTyoe) {
```

Comment:
"Type" in both actual parameter and type misspelled.

Reply:
ok